frontend/public/kubevirt/components/cluster/cluster-overview.jsx

```diff
@@ -167,27 +169,37 @@ export class ClusterOverview extends React.Component {
   componentDidMount() {
     this._isMounted = true;
 
-    this.fetchPrometheusQuery(CONSUMERS_CPU_QUERY, 'workloadCpuResults');
-    this.fetchPrometheusQuery(CONSUMERS_MEMORY_QUERY, 'workloadMemoryResults');
-    this.fetchPrometheusQuery(NODE_CONSUMERS_MEMORY_QUERY, 'infraMemoryResults');
-    this.fetchPrometheusQuery(NODE_CONSUMERS_CPU_QUERY, 'infraCpuResults');
-
     this.fetchHealth();
 
-    this.fetchPrometheusQuery(OPENSHIFT_VERSION_QUERY, 'openshiftClusterVersionResponse');
+    if (getPrometheusBaseURL()){
+      this.fetchPrometheusQuery(CONSUMERS_CPU_QUERY, 'workloadCpuResults');
+      this.fetchPrometheusQuery(CONSUMERS_MEMORY_QUERY, 'workloadMemoryResults');
+      this.fetchPrometheusQuery(NODE_CONSUMERS_MEMORY_QUERY, 'infraMemoryResults');
+      this.fetchPrometheusQuery(NODE_CONSUMERS_CPU_QUERY, 'infraCpuResults');
+
+      this.fetchPrometheusQuery(OPENSHIFT_VERSION_QUERY, 'openshiftClusterVersionResponse');
+
+      this.fetchPrometheusQuery(CAPACITY_MEMORY_TOTAL_QUERY, 'memoryTotal');
+      this.fetchPrometheusQuery(CAPACITY_NETWORK_TOTAL_QUERY, 'networkTotal');
+      this.fetchPrometheusQuery(CAPACITY_NETWORK_USED_QUERY, 'networkUsed');
 
-    this.fetchPrometheusQuery(CAPACITY_MEMORY_TOTAL_QUERY, 'memoryTotal');
-    this.fetchPrometheusQuery(CAPACITY_NETWORK_TOTAL_QUERY, 'networkTotal');
-    this.fetchPrometheusQuery(CAPACITY_NETWORK_USED_QUERY, 'networkUsed');
+      this.fetchPrometheusQuery(CEPH_OSD_UP_QUERY, 'cephOsdUp');
+      this.fetchPrometheusQuery(CEPH_OSD_DOWN_QUERY, 'cephOsdDown');
 
-    this.fetchPrometheusQuery(CEPH_OSD_UP_QUERY, 'cephOsdUp');
-    this.fetchPrometheusQuery(CEPH_OSD_DOWN_QUERY, 'cephOsdDown');
+      this.getStorageMetrics();
 
-    this.getStorageMetrics();
+      this.fetchPrometheusQuery(UTILIZATION_CPU_USED_QUERY, 'cpuUtilization');
+      this.fetchPrometheusQuery(UTILIZATION_MEMORY_USED_QUERY, 'memoryUtilization');
+    } else {
+      warn('Prometheus BASE URL is missing!');
```

Comment:
prometheus results will remain undefined which will result in loading states for cards which expect them. I'd expect that the cards will indicate not available state

Reply:
yup, my bad! I will take a look at it.

Reply:
fixed. The cards will show not available now. Maybe we should ask designers to show that Prometheus/AlertManager  is available in the health card as a followup?

Reply:
@suomiy - You mean work in a way to tell the user that Prometheus is unavailable, right?

What do you think @andybraren?

Reply:
yes, + I guess also AlertManager could be unavailable - we might also work better on the detection

Reply:
We have an upcoming story to reassess how the Health card works, so this is good timing. Is something like this roughly what you’d expect to see?

![health-prometheus](https://user-images.githubusercontent.com/9122899/57697549-84788500-7621-11e9-84d3-90f10eebf93a.png)

Reply:
+1 looks good. 

We might also distinguish between Prometheus and AlertManager - both belong to monitoring. 
And between states not available and down (the pods for each on of them could crash).